integration-test/fire-button.spec.js

```diff
@@ -134,7 +134,7 @@ test.describe('Fire Button', () => {
                 name: 'Last7days',
                 descriptionStats: {
                     clearHistory: true,
-                    cookies: 3, // gets the number of domains with cookies set
+                    cookies: 2, // gets the number of domains with cookies set
```

Comment:
@sammacbeth I'm not sure why this is necessary, seems to be unrelated the changes here?